src/languages/swift.js

```diff
@@ -283,6 +283,49 @@ export default function(hljs) {
   };
   TYPE.contains.push(GENERIC_ARGUMENTS);
 
+  // https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID380
+  const OPERATOR_DECL = {
+    beginKeywords: 'operator',
+    contains: [
+      {
+        match: /\s+/,
+        relevance: 0
+      },
```

Comment:
```suggestion
  const OPERATOR_DECL = {
    beginKeywords: 'operator',
    ends: hljs.NEVER,
    contains: [
      {
```

I'm not sure what to name it (never isn't quite right) but for things like this rather than all that muck to eat spaces it should be easier to just write "leave the mode open until you find a token"...

Reply:
I'd also be open to a new attribute to suggest this if it was clear and easy to read.  But it would really just be sugar for the above.

Reply:
Is `MATCH_NEVER` just a regex that never matches, such as `\b\B`? I guess it's a bit confusing because it makes it seems as though the mode never ends. But I assume you'd have to combine this with a child mode that ends its parent?

Reply:
> Is MATCH_NEVER just a regex that never matches, such as \b\B? 

Well, it doesn't exist yet, but we'd add it - and yes, exactly.

> I guess it's a bit confusing because it makes it seems as though the mode never ends. 

Exactly, *unless manually ended by a child*... obviously the mode ends upon input termination.  My concern is that confusion between "never ends, literally" and "never ends, but by child".

> But I assume you'd have to combine this with a child mode that ends its parent?

Yes, as you already have... this would just avoid that extraneous parsing of whitespace.

I very much thing the approach is correct, and obviously we could do it with just `\b\B` and a comment, but it seems nice to bless the pattern with a proper name. @allejo Any ideas?